src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -125,22 +125,34 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
        
         private HttpContent GetBody(string referenceId, bool schemaLessBody, Dictionary<string, (OpenApiSchema Schema, FormulaValue Value)> map)
         {
-            FormulaValueSerializer serializer = _argMapper.ContentType.ToLowerInvariant() switch
-            {
-                OpenApiExtensions.ContentType_XWwwFormUrlEncoded => new OpenApiFormUrlEncoder(schemaLessBody),                
-                OpenApiExtensions.ContentType_TextPlain => new OpenApiTextSerializer(schemaLessBody),
-                _ => new OpenApiJsonSerializer(schemaLessBody)
-            };
-            
-            serializer.StartSerialization(referenceId);
-            foreach (var kv in map)
+            FormulaValueSerializer serializer = null;
```

Comment:
Is this related to the `[]` syntax change? If not, can this be a separate PR?

Reply:
I'll create a separated PR then